requirements.txt

```
@@ -1,2 +1,3 @@
 Appium-Python-Client==2.6.1;python_version >= '3.0'
-browserstack-local==1.2.2
\ No newline at end of file
+browserstack-local==1.2.2
+browserstack-sdk==1.4.0
```

let's not pin the version.. should always pick latest
```suggestion
browserstack-sdk
```

---

done